src/store/actions.ts

```diff
@@ -190,14 +190,18 @@ export default {
     if (state.order.orderNumber) {
       formData.user = state.order.user
       formData.email = state.order.email
+      const isapplicationFormUpdate = formData.applicationForm && (typeof formData.applicationForm.filename !== 'string')
 
       await updateOrder(formData, formFields)
 
-      // Assume admin edits the user's order.
+      // If a admin edits the order, put back permissions
```

Comment:
Comments vind ik persoonlijk wat beter leesbaar als het begint met de actie, bijv:
// Put back permissions when a**n** admin edits the order.

Reply:
The comment is related to the line below, this is a existing comment i updated to reflect the code changes i made